### PR TITLE
Add rubocop-capybara to suggested extensions and extension doc

### PR DESCRIPTION
**Pull request by ydah**
This PR adds [rubocop-capybara](https://github.com/rubocop/rubocop-capybara) to the `SuggestExtensions` list and extension doc.

-----------------

Before submitting the PR make sure the following are checked:

* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Wrote [good commit messages][1].
* [-] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [-] Added tests.
* [x] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [x] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by koic** [comment]
Can you update the extension doc?
https://github.com/rubocop/rubocop/blob/master/docs/modules/ROOT/pages/extensions.adoc#official-extensions

File: docs/modules/ROOT/pages/extensions.adoc [hunk 1]
```diff
@@ -84,6 +84,8 @@ Rake-specific analysis
 Code style checking for Sequel gem
 * https://github.com/rubocop/rubocop-thread_safety[rubocop-thread_safety] -
 Thread-safety analysis
+Capybara-specific analysis
+* https://github.com/rubocop/rubocop-capybara[rubocop-capybara] -
```
Comment (koic) [inline docs/modules/ROOT/pages/extensions.adoc:88]:
```suggestion
* https://github.com/rubocop/rubocop-capybara[rubocop-capybara] -
Capybara-specific analysis
```

**Comment by ydah** [comment]
I see that the extension doc was not updated either. Thank you very much. I updated extension doc.

File: docs/modules/ROOT/pages/extensions.adoc (hunk 1 above)
Comment (ydah) [reply docs/modules/ROOT/pages/extensions.adoc]: Oh sorry😓 I updated this PR.

**Pull request merged**